integration-tests/src/main/java/io/druid/testing/clients/EventReceiverFirehoseTestClient.java

```diff
@@ -150,12 +151,12 @@ public int postEventsFromFile(String file)
                            ? MediaType.APPLICATION_JSON
                            : SmileMediaTypes.APPLICATION_JACKSON_SMILE;
         totalEventsPosted += postEvents(events, mapper, mediaType);
-        ;
+        expectedEvents += events.size();
```

Comment:
not related to this PR, it was an integration-test issue, that popped up when i tried to run integration-tests. will separate test fix in a separate PR.